programs/zstdcli.c

```diff
@@ -118,6 +118,7 @@ static int usage(const char* programName)
 #endif
     DISPLAY( " -D file: use `file` as Dictionary \n");
     DISPLAY( " -o file: result stored into `file` (only if 1 input file) \n");
+    DISPLAY( "--exclude-compressed:  only compress files that are not previously compressed \n");
```

Comment:
That's nice to have it in the `-h` screen,
though this rather belong to the "advanced" usage category.

Reply:
Addressed in latest commit "Fixing tests and moving flag to advanced usage 0e9a37d"